nix/tests.nix

```diff
@@ -78,7 +79,7 @@ let
       name
       ''
         env="$(cat ${postgrest.env})"
-        export PATH="$env/bin:${curl}/bin:$PATH"
+        export PATH="$env/bin:${curl}/bin:${procps}/bin:$PATH"
```

Comment:
just a fix, I forgot yesterday. Only affects running io tests in `nix-shell --pure`.